src/routes/cache-hooks/cache-hooks.controller.spec.ts

```diff
@@ -207,9 +208,13 @@ describe('Post Hook Events (Unit)', () => {
     },
   ])('$type clears balances', async (payload) => {
     const safeAddress = faker.finance.ethereumAddress();
-    const chainId = faker.string.numeric();
+    const chainId = faker.string.numeric({
+      exclude: configurationService.getOrThrow(
+        'features.zerionBalancesChainIds',
+      ),
+    });
```

Comment:
I had to exclude the Zerion-configured chains, otherwise, the cache key wouldn't match (`*_safe_balances_*` vs. `*_zerion_balances_*`). 

I think It feels a bit weird to manage this cache-specific stuff from a controller test, but also I think this considerations are out of the scope of the PR.